lib/tasks/import.rake

```diff
@@ -7,14 +7,18 @@ namespace :import do
     CSV.foreach(file, headers: true) do |row|
       Customer.create!(row.to_hash)
     end
+    puts "Imported Customers!"
   end
 
   desc "Import invoice items from csv"
   task invoice_items: :environment do
     file = File.join Rails.root, "db/data/invoice_items.csv"
     CSV.foreach(file, headers: true) do |row|
-      InvoiceItem.create!(row.to_hash)
+      unit_price = row["unit_price"].to_f/100.00
```

Comment:
@concach Yay! You figured this out! I gave up on it yesterday after having all those struggles with `rake import`, so I'm glad you got it. Good work